kodilla-patterns/src/test/java/com/kodilla/patterns/factory/tasks/TaskFactoryTestSuite.java

```diff
@@ -0,0 +1,66 @@
+package com.kodilla.patterns.factory.tasks;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TaskFactoryTestSuite {
+
+    private TaskFactory taskFactory = new TaskFactory();
+    private Task shoppingTask = taskFactory.createTask(TaskFactory.SHOPPINGTASK);
+    private Task paintingTask = taskFactory.createTask(TaskFactory.PAINTINGTASK);
+    private Task drivingTask = taskFactory.createTask(TaskFactory.DRIVINGTASK);
+
+    @Test
+    public void testCreateShoppingTask() {
+
+        //Then
```

Comment:
W każdej z metod stosujmy podział `given when then`. Jeśli wszystko sie sprowadza do jednej sekcji, to wówczas piszemy `// given when then`